readme.md

```diff
@@ -58,105 +62,126 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 * **Your Answer:**
 
----
+## The response header contains the date, size and type of file that the server is sending back to the client and also data about the server itself.
 
 - [ ] Add the following line **above** the `res.json()` line in your `app.js` file; then, make the request again.
   ```js
-  console.log(req.headers)
+  console.log(req.headers);
   ```
 
 * **Question:** Where can you find this `console.log()` statement? How can you change the headers that are sent in Postman?
 
 * **Your Answer:**
 
----
+## console.log() statement can be find in the terminal.
+
+## By adding new key:value under the headers tab in postman we can have custom information.
 
 - [ ] Add the following line **above** the `res.json()` line in your `app.js` file; then, make a request again but this time change the URL to the following: `http://localhost:5000?course=javascript`
   ```js
-  console.log(req.query)
+  console.log(req.query);
   ```
 
 * **Question:** What are query parameters? Try going to a couple of your favorite websites and paste an example of query parameters being used.
 
 * **Your Answer:**
 
----
+## Query parameters are used to make our request more specific. In this case we send them as a key:value pairs.
 
 - [ ] Before all of your routes, add the following:
   ```js
   app.use((req, res, next) => {
-    console.log('In the server!')
-    next()
-  })
+    console.log("In the server!");
+    next();
+  });
   ```
 
 * **Question:** When does `app.use()` get called?
 
 * **Your Answer:**
 
----
+## app.use() is a general function for every request to go through
```

Comment:
Remember each request moves down the list until it finds an endpoint and will only continue if next is called inside a function.